### PR TITLE
Add Altair and Basilisk chains

### PR DESCRIPTION
**Pull request by ERussel**

### CODE REVIEW

File: chains/v2/chains.json [hunk 1]
```diff
@@ -436,6 +436,58 @@
...
+            {
+                "assetId": 0,
+                "symbol": "BSX",
+                "precision": 12
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://rpc-01.basilisk.hydradx.io",
+                "name": "Hydradx"
```
Comment (antonkhvorov) [inline chains/v2/chains.json:453]: HydraDX node

File: chains/v2/chains.json [hunk 2]
```diff
@@ -436,6 +436,58 @@
...
+            {
+                "assetId": 0,
+                "symbol": "AIR",
+                "precision": 18
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://fullnode.altair.centrifuge.io",
+                "name": "Centrifuge"
```
Comment (antonkhvorov) [inline chains/v2/chains.json]: Centrifuge node

File: chains/v2/chains.json [hunk 3]
```diff
@@ -436,6 +436,58 @@
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Bifrost.svg",
         "addressPrefix": 6
     },
+    {
+        "chainId": "a85cfb9b9fd4d622a5b28289a02347af987d8f73fa3108450e2b4a11c1ce5755",
```
Comment (antonkhvorov) [inline chains/v2/chains.json:440]: a85cfb9b9fd4d622a5b28289a02347af987d8f73fa3108450e2b4a11c1ce5755
verified

File: chains/v2/chains.json (hunk 1 above)
Comment (ERussel) [reply chains/v2/chains.json]: fixed

File: chains/v2/chains.json [hunk 4]
```diff
@@ -436,6 +436,58 @@
...
+            {
+                "url": "wss://basilisk.api.onfinality.io/public-ws",
+                "name": "OnFinality node"
+            }
+        ],
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Basilisk.svg",
+        "addressPrefix": 10041
+    },
+    {
+        "chainId": "aa3876c1dc8a1afcc2e9a685a49ff7704cfd36ad8c90bf2702b9d1b00cc40011",
```
Comment (antonkhvorov) [inline chains/v2/chains.json:464]: aa3876c1dc8a1afcc2e9a685a49ff7704cfd36ad8c90bf2702b9d1b00cc40011
verified

File: chains/v2/chains.json (hunk 2 above)
Comment (ERussel) [reply chains/v2/chains.json]: fixed

**Comment by valentunn** [comment]
# Basilisk  ✔️

## Nodes ✔️ 

## Genesis ✔️ 
0xa85cfb9b9fd4d622a5b28289a02347af987d8f73fa3108450e2b4a11c1ce5755

image

## Chain properties ✔️ 

ss58 format is not defined on-chain

image

## Android App v2.1.0 release working ✔️ 

image


# Altair ✔️

## Nodes ✔️ 

## Genesis ✔️ 
0xaa3876c1dc8a1afcc2e9a685a49ff7704cfd36ad8c90bf2702b9d1b00cc40011

image

## Chain properties ✔️ 

ss58 format is not defined on-chain

image


## Android App v2.1.0 release working ✔️ 

image

**Comment by valentunn** [comment]
LGTM

**Review by antonkhvorov — APPROVED** [review]

**Pull request merged**